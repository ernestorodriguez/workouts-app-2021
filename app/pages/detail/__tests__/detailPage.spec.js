/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { shallow } from 'enzyme';
import reactRouter from 'react-router';
import Page from '../index';

jest.mock('react-router');

describe('Detail page', () => {
  test('should match snapshot', () => {
    global.location = '/';
    reactRouter.useParams.mockResolvedValue({ alias: 'ALIAS MOCK' });
    const component = (
      <Page />
    );
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
  });
});
