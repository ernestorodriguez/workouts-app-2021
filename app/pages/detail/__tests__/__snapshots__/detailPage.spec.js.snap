// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail page should match snapshot 1`] = `
<div
  data-js="details-page"
>
  <Header
    internal={true}
  />
  <ItemDetail />
</div>
`;
